packages/statsd/src/client.ts

```diff
@@ -0,0 +1,127 @@
+import {StatsD, ClientOptions, Tags} from 'hot-shots';
+import {snakeCase} from 'change-case';
+
+export interface Timer {
+  stop(): number;
+}
+
+export interface Logger {
+  log(message: string): void;
+}
+
+export interface Options extends ClientOptions {
+  logger?: Logger;
+  snakeCase?: boolean;
+}
+
+export class StatsDClient {
+  private statsd: StatsD;
+  private logger: Logger | null = null;
```

Comment:
can we default this to a dirt simple object that just exposes console.log when there is no logger provided?